mechanicalsoup/browser.py

```diff
@@ -56,26 +74,32 @@ def set_user_agent(self, user_agent):
         self.session.headers['User-agent'] = user_agent
 
     def request(self, *args, **kwargs):
-        """Straightforward wrapper around requests session.request
-        (http://docs.python-requests.org/en/master/api/).
+        """Straightforward wrapper around `requests.Session.request
+        <http://docs.python-requests.org/en/master/api/#requests.Session.request>`__.
 
         This is a low-level function that should not be called for
-        basic usage (use .get or .post instead). Use it if you need an
-        HTTP verb that mechanicalsoup doesn't manage (e.g. MKCOL) for
+        basic usage (use :func:`get` or :func:`post` instead). Use it if you
+        need an HTTP verb that MechanicalSoup doesn't manage (e.g. MKCOL) for
         example.
         """
         response = self.session.request(*args, **kwargs)
         Browser.add_soup(response, self.soup_config)
         return response
 
     def get(self, *args, **kwargs):
+        """Straightforward wrapper around `requests.Session.get
+        <http://docs.python-requests.org/en/master/api/#requests.Session.get>`__.
```

Comment:
Probably this should mention that in addition to requests's get, we remember the soup object in the browser.

Reply:
Agreed! I've added more complete return type information to the Browser methods that return a Response+Soup.